cli/flags/flags.go

```diff
@@ -34,6 +34,7 @@ const (
 	ethKeystorePath   = "ethKeystorePath"
 	ethKeystorePass   = "ethKeystorePass"
 	ethEndpointURL    = "ethEndpointURL"
+	signedReshare     = "signedReshare"
```

Comment:
This is not used?

Reply:
not yet, I think we will create additional flags at bulk?

Reply:
imo we should have `bulkReshareJSON` and `bulkResignJSON` flags which can accept filepath or raw JSON with the structure like here https://github.com/ssvlabs/ssv-dkg/blob/gnosis_multisig/integration_test/stubs/reshare/bulk_reshare_msgs.json